app/src/main/res/menu/main.xml

```diff
@@ -2,17 +2,35 @@
 <menu xmlns:android="http://schemas.android.com/apk/res/android" >
     
     <!-- TODO: add icons -->
+    <item
```

Comment:
As we're adding more and more items to the menu, maybe we should use icons instead to save space.

Reply:
I've seen the TODO yes, I'll have a look how to do it.